internal/rpcserver/rpcserverhandlers_test.go

```diff
@@ -1239,7 +1239,6 @@ func defaultMockConfig(chainParams *chaincfg.Params) *Config {
 		ChainParams:     chainParams,
 		Chain:           defaultMockRPCChain(),
 		SanityChecker:   defaultMockSanityChecker(),
-		BlockTemplater:  defaultMockBlockTemplater(),
```

Comment:
Looks like a rebase merge mistake.  This shouldn't be removed.  All the others adding the default back in should be removed instead.

Reply:
I particularly removed it. This it'd be assumed that the server always starts in mining mode with a block templater set. This interferes with `"handleRegenTemplate: node is not configured for mining"` because you'd have to pass a nil block templater to the config which we are currently preventing for mock config options.

Reply:
I see what you're saying.  This was handled in a difference case by having a flag that specifically said you don't want it (`setExistsAddresserNil` iirc). 

I think it's important to have one set by default and then override it to nil, because, otherwise, it will very likely lead to strange test failures in the future due to it being nil in the default case.  It's almost always better in tests to start with a base that fully works and then take things away rather than the other way around, because people that come along later have no idea that you need to add some magical thing to make it work.

Reply:
Gotcha, updating.